{{ cookiecutter.package_name|slugify }}/Dockerfile

```diff
@@ -73,6 +73,10 @@ COPY . .
 
 # Expose the application.
 {%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
+
+RUN --mount=type=cache,target=/root/.cache \
+    pip install --no-input poethepoet
```

Comment:
Unfortunately, `poe` is not available in the App image.
I tracked down where the issue starts to arise. In the ci layer, when we run `poetry install --no-dev --no-interaction`, it deletes `poe` but not the other binaries in  `/usr/local/bin` such as (pip and starship). It's because `poethepoet` is only a dev dependency, so poetry doesn't see it with `--no-dev` flag and decides to remove that package.
The simplest way to fix the issue is to move the poetry dependency from dev to main dependencies. It's done now and I removed the need to reinstall poethepoet

Reply:
I see, thanks for investigating! For posterity, this is the relevant Poetry issue: https://github.com/python-poetry/poetry/issues/3139.